firebase-functions/src/main/java/com/google/firebase/functions/FirebaseContextProvider.java

```diff
@@ -62,9 +63,9 @@ class FirebaseContextProvider implements ContextProvider {
   }
 
   @Override
-  public Task<HttpsCallableContext> getContext() {
+  public Task<HttpsCallableContext> getContext(boolean limitedUseAppCheckToken) {
```

Comment:
I thought about using a different name internally because agreed it's confusing, but I decided to stick with the external-facing name to make it clear where this is coming from.